hooks/pre-command

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "~~~ Installing nvm"
+
+NVM_DIR="$(mktemp -d)"
+export NVM_DIR
+touch "$NVM_DIR/.automattic-nvm-dir-marker"
+
+# Store the installation directory in meta-data so that we can uninstall it later in the pre-exit hook.
+buildkite-agent meta-data set "automattic-nvm-installation-dir" "$NVM_DIR"
+echo "Installing nvm in $NVM_DIR"
+
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.4/install.sh | PROFILE=/dev/null bash
```

Comment:
💡  Wondering if we should extract the version of the `nvm` tool (`v0.39.4`) into a constant to facilitate updating the plugin when new versions of the `nvm` tool will be released? 🤷

Reply:
I thought about that. It's probably a bit hard for this plugin to promise it'll work on _all_ nvm versions. Plus, updating nvm version inside the plugin means we can test all this plugin properly for each release.

Reply:
To be clear I was not suggesting to make the version of nvm a plugin parameter. Just move the version in a **constant** (eg `NVM_VERSION=0.39.2` at the top of this hook / bash script, then use `$NVM_VERSION` in the URL) so that when we decide to update the nvm version used by the plugin in the future it'd make it easier to make a PR that updates that constant in this repo the create a new release of the plugin.

But this was very nitpicky comment and the benefit is negligeable so you can also ignore my comment 😛